hypothesis-python/tests/cover/test_direct_strategies.py

```diff
@@ -424,6 +420,41 @@ def test_average_size_is_deprecated():
     ds.lists(ds.integers(), average_size=1).example()
 
 
+@checks_deprecated_behaviour
+@pytest.mark.parametrize('fn,kwargs', [
+    (ds.integers, {'min_value': decimal.Decimal('1.5')}),
+    (ds.integers, {'max_value': decimal.Decimal('1.5')}),
+    (ds.integers, {'min_value': -1.5, 'max_value': -0.5}),
+    (ds.floats, {'min_value': 1.8, 'width': 16}),
+    (ds.floats, {'max_value': 1.8, 'width': 16}),
```

Comment:
[`width=16` is only supported on Python 3.6+ or with Numpy](https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.floats), but you could just use `width=32`.